utils/hash.ts

```diff
@@ -1,7 +1,16 @@
 import * as crypto from "crypto";
+import { FamilyMembers } from "../generated/definitions/FamilyMembers";
 
 export const toHash = (s: string): string => {
   const hash = crypto.createHash("sha256");
   hash.update(s);
   return hash.digest("hex");
 };
+
+export const generateFamilyUID = (familyMembers: FamilyMembers): string =>
```

Comment:
```suggestion
export const generateFamilyUID = (familyMembers: FamilyMembers): NonEmptyString =>
```
By enforcing this, you may have to handle the edge case of `familyMembers = []`

Reply:
Maybe with an `Either<Error, NonEmptyString>`?

Reply:
it's ok to return the hash in case of [] since the INPS webservice won't never return an empty array